packages/core/package.nls.json

```
@@ -9,6 +9,7 @@
     "AWS.configuration.description.lambda.recentlyUploaded": "Recently selected Lambda upload targets.",
     "AWS.configuration.description.ecs.openTerminalCommand": "The command to run when starting a new interactive terminal session.",
     "AWS.configuration.description.iot.maxItemsPerPage": "Controls how many IoT Things, Certificates, or Policies are listed before showing a node to `Load More...`.",
+    "AWS.configuration.description.dynamodb.maxItemsPerPage": "Controls how many items to show per page in dynamoDb table view.",
```

I think this should be:
```suggestion
    "AWS.configuration.description.dynamodb.maxItemsPerPage": "Controls how many items to show per page in dynamoDB table view.",
```

or 
```suggestion
    "AWS.configuration.description.dynamodb.maxItemsPerPage": "Controls how many items to show per page in DynamoDB table view.",
```